TurfRegressionTests/SecondaryIndexingRegressionTests.swift

```diff
@@ -1,9 +1,63 @@
+import XCTest
+
+class SecondaryIndexingRegressionTests: XCTestCase {
```

Comment:
You might have to rebase/merge master into your branch. I've added Quick and Nimble to write these tests.

You should be able to follow the outline of the Basic regression tests.